modules/azure/vnet/outputs.tf

```diff
@@ -9,3 +9,37 @@ output "master_subnet" {
 output "worker_subnet" {
   value = "${var.external_vnet_name == "" ?  join(" ", azurerm_subnet.worker_subnet.*.id) : var.external_worker_subnet_id }"
 }
+
+# TODO: Allow user to provide their own network
+output "etcd_cidr" {
+  value = "${azurerm_subnet.master_subnet.address_prefix}"
+}
+
+# TODO: Allow user to provide their own network
+output "master_cidr" {
+  value = "${azurerm_subnet.master_subnet.address_prefix}"
+}
+
+# TODO: Allow user to provide their own network
+output "worker_cidr" {
+  value = "${azurerm_subnet.worker_subnet.address_prefix}"
+}
+
+# TODO: Allow user to provide their own network
+output "admin_cidr" {
```

Comment:
This is not used and causes confusion. We should prob remove it and all refs of it